src/ansys/dpf/core/operators/logic/elementary_data_selector.py

```diff
@@ -238,6 +263,27 @@ def default_value(self):
         """
         return self._default_value
 
+    @property
+    def elementary_data_index_2(self):
+        """Allows to connect elementary_data_index_2 input to the operator.
+
+        For field with nature matrix, this is the
+        column indices to extract.
```

Comment:
```suggestion
        """Elementary data indices that are inputs to the operator.
```
A description for a property should be presented as a noun phrase. I am fairly certain that the "Parameters" section should be removed.